firebase-firestore/ktx/ktx.gradle

```diff
@@ -45,6 +46,19 @@ android {
     }
 }
 
+android.libraryVariants.all { variant ->
```

Comment:
why is this needed?

Reply:
I think this is a must for integration test. Without this, integration test will not actually run, and show "0 passed".
BTW, during the run of a integration test, the first line printed in the "Test Results" console is:

`$ adb shell am instrument -w -m    -e debug false -e class 'com.google.firebase.firestore.NestedMapEncoderIntegrationTest' com.google.firebase.firestore.ktx.test/androidx.test.runner.AndroidJUnitRunner`